src/services/coreBackService.js

```diff
@@ -3,42 +3,52 @@ const {
   API_CRI_CALLBACK,
   API_SESSION_INITIALISE,
   API_BASE_URL,
+  API_JOURNEY_EVENT,
 } = require("../lib/config");
 
-const {
-  generateAxiosConfig,
-  generateAxiosConfigWithClientSessionId,
-  generateJsonAxiosConfig,
-  createAxiosInstance,
-} = require("../app/shared/axiosHelper");
+const { createAxiosInstance } = require("../app/shared/axiosHelper");
 
 const axiosInstance = createAxiosInstance(API_BASE_URL);
 
-function postAction(req, action) {
+function generateAxiosConfig(req) {
```

Comment:
One small consequence of this - we will now send the two session ids to all endpoints (including `initialiseIpvSession`) if they are available. I don't think this is a big deal, but we could explicitly control it if we wish.

Reply:
Why move this out of axiosHelper?